src/common/utils.ts

```diff
@@ -278,5 +278,5 @@ export const getCurrencySymbol = ({
     .trim();
 };
 
-export const concatText = (a?: string, b?: string) =>
-  [a, b].filter(Boolean).join(`\n`);
+export const concatTextToNewline = (...args: Array<string | undefined>) =>
```

Comment:
now we are in the rabbit whole, but could have also accept separator 😆 

{
   strings: [...]
   separator: '\n'
}

Reply:
I renamed it to `concatTextToNewline` so it only concats to newline

Reply:
I am just against this kind of helpers widespread use, they don't add anything else and often need to be expanded when new use arises randomly. It is what it is I guess.